google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ModelInfo.java

```diff
@@ -91,12 +92,14 @@ public Model apply(ModelInfo ModelInfo) {
     /**
      * Set the labels applied to this model.
      *
-     * <p>When used with {@link BigQuery#update(ModelInfo, ModelOption...)}, setting {@code labels}
-     * to {@code null} removes all labels; otherwise all keys that are mapped to {@code null} values
-     * are removed and other keys are updated to their respective values.
+     * <p>When used with {@link BigQuery#update(ModelInfo, BigQuery.ModelOption...)}, setting {@code
```

Comment:
just a minor nit: change this `{@link BigQuery#update(ModelInfo, BigQuery.ModelOption...)}` => `{@link BigQuery#update(ModelInfo, ModelOption...)}`

By doing that you won't unnecessarily change these three lines